### DISCUSSION TITLE
Continously enter values in multiselect

### DISCUSSION THREAD

**Discussion by xoitx** [mantinedev/mantine#4221] [Q&A]
How to get the cursor to continuously accept input after selection.
So if I select an option, then the cursor should be ready for the next option and dropdown should not close.

`import { useEffect, useState } from 'react';
import { ActionIcon, Badge, Flex, MultiSelect, MultiSelectValueProps, rem, TextInput } from '@mantine/core';
import keywordOptions from '../KeywordsEntry/keywordOptions'
import { IconX } from '@tabler/icons-react';

export default function KeywordsEntry() {
    const [keywordList, setKeywordList] = useState<{ label: string, value: string }[]>(keywordOptions)
    const [keywordsSelected, setKeywordsSelected] = useState([]) // initialize it as an empty array

    const handleChange = (change: any) => {
        setKeywordsSelected(change)
    }

    const value = ({ label, onRemove, ...others }: MultiSelectValueProps) => {
        const removeButton = (
            <ActionIcon onMouseDown={onRemove} size="xs" color="blue" radius="xl" variant="transparent">
                <IconX size={rem(10)} />
            </ActionIcon>
        );
        return <div {...others}>
            <Badge key={label} size="sm" variant="outline" rightSection={removeButton}>
                {label}
            </Badge>
        </div>
    }

    const MultiSelectComponent = () => (
        <MultiSelect
            hoverOnSearchChange
            radius={'md'}
            label={`Keywords (${keywordsSelected.length})`}
            value={keywordsSelected}
            data={keywordList}
            searchable
            creatable
            getCreateLabel={(query) => `+ Create ${query}`}
            onChange={handleChange}
            placeholder={"Enter customer's keywords"}
            clearable
            filter={(value, selected, item) => {
                if (!selected && item.label?.toLowerCase().startsWith(value.toLowerCase().trim())) {
                    return true
                }
                else return false
            }
            }
            onCreate={(query) => {
                const item = { value: query, label: query };
                setKeywordList((current: any) => [...current, item]);
                return item;
            }}
            styles={{
                input: { minHeight: '7rem', backgroundColor: '#FFFFFF00', },
            }}
            valueComponent={value}
            transitionProps={{ duration: 150, transition: 'pop-top-left', timingFunction: 'ease' }}
        />
    )

    return (
        <>
            <div style={{ display: 'flex', flexDirection: 'column', justifyContent: 'space-around' }}>
                <MultiSelectComponent />
            </div>
        </>
    );
}`

**Comment by yverby** [comment]
You can try the following
https://codesandbox.io/s/nifty-knuth-bg4yjc?file=/src/App.tsx

**Comment by 7iomka** [comment]
Hm
Seems like default example in the docs works the same way
https://mantine.dev/core/multi-select/#searchable
